zuessite/templates/local_settings_template.py

```diff
@@ -1,23 +1,20 @@
 # Local settings for Zues.
+# Automatically generated
 
-# Step 1: generate a secret key and set it here.
-# For example, run 'pwgen -s 50 1' and set the result here.
-# Do not reuse a secret key from another place.
-SECRET_KEY = ''
+SECRET_KEY = '{{ secret_key }}'
 
-# Debug should be False in production
-DEBUG = False
+# Debug should be False in production.
+DEBUG = True
 
-# Step 2: put all the domain names under which the site should be reachable
-# in this list.
-ALLOWED_HOSTS = []
+# All the domain names under which the site should be reachable.
+ALLOWED_HOSTS = ["0.0.0.0"]
```

Comment:
We should revert this as ALLOWED_HOSTS is not used in development and should never template all hostnames.